src/EmbedIO/Security/BannedInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Net;
+
+namespace EmbedIO.Security
+{
+    /// <summary>
+    /// Represents the info af a banned IP address.
+    /// </summary>
+    public class BannedInfo
+    {
+        /// <summary>
+        /// Gets or sets the banned IP address.
+        /// </summary>
+        public IPAddress IPAddress { get; set; }
+
+        /// <summary>
+        /// Gets or sets until when the IP will remain ban.
+        /// </summary>
+        public long BanUntil { get; set; }
```

Comment:
If this property represents the time of expiration, it could be named `ExpiresAt` and be of type `DateTime`.
If it represents the duration of the ban, `Duration` would be a more sensible name, and `TimeSpan` a more appropriate type - unless it is measured in minutes, in which case `long` is fine, but the name could be changed to `DurationMinutes`.

Reply:
This property represents the time of expiration, I already renamed as ´ExpiresAt´. We use `long` only for comparison purposes.